docs/source/daml-finance-rst/Daml-Finance-Interface-Settlement-Settleable.rst

```diff
@@ -110,19 +110,19 @@ Data Types
        * - steps
          - \[:ref:`Step <type-daml-finance-interface-settlement-types-step-78661>`\]
          - Settlement steps\.
-
+  
   **instance** `Eq <https://docs.daml.com/daml/stdlib/Prelude.html#class-ghc-classes-eq-22713>`_ `View <type-daml-finance-interface-settlement-settleable-view-20035_>`_
-
+  
   **instance** `Show <https://docs.daml.com/daml/stdlib/Prelude.html#class-ghc-show-show-65360>`_ `View <type-daml-finance-interface-settlement-settleable-view-20035_>`_
-
-  **instance** HasInterfaceView `Settleable <type-daml-finance-interface-settlement-settleable-settleable-40815_>`_ `View <type-daml-finance-interface-settlement-settleable-view-20035_>`_
-
+  
+  **instance** `HasInterfaceView <https://docs.daml.com/daml/stdlib/Prelude.html#class-da-internal-interface-hasinterfaceview-4492>`_ `Settleable <type-daml-finance-interface-settlement-settleable-settleable-40815_>`_ `View <type-daml-finance-interface-settlement-settleable-view-20035_>`_
+  
   **instance** (HasIsInterfaceType t, `HasTemplateTypeRep <https://docs.daml.com/daml/stdlib/Prelude.html#class-da-internal-template-functions-hastemplatetyperep-24134>`_ t, `Implements <https://docs.daml.com/daml/stdlib/Prelude.html#type-da-internal-interface-implements-92077>`_ t `Settleable <type-daml-finance-interface-settlement-settleable-settleable-40815_>`_) \=\> `HasExercise <https://docs.daml.com/daml/stdlib/Prelude.html#class-da-internal-template-functions-hasexercise-70422>`_ t GetView `View <type-daml-finance-interface-settlement-settleable-view-20035_>`_
```

Comment:
Seems more keywords are clickable now, but not `HasIsInterfaceType` yet. Maybe not part of this PR though?

Reply:
mm, not sure why it is the case. I think we probably want to remove those instances anyway. I will try it out as part of this PR directly to see how it looks like.